packages/shared/src/__tests__/WithNativeNavigation.test.js

```diff
@@ -0,0 +1,116 @@
+/* @flow */
+
+import * as React from 'react';
+import { View } from 'react-native';
+import renderer from 'react-test-renderer';
+import { GestureController } from '@kiwicom/mobile-shared';
+
+import WithNativeNavigation from '../WithNativeNavigation';
+
+describe('WithNativeNavigation', () => {
+  const WrappedComponent = () => <View />;
+  const TestKey = 'test-key';
+  const getComponent = () => {
+    // $FlowFixMe injected props problem
```

Comment:
can you please `$FlowExpectedError:` instead, and I don't understand the text. What is the problem with the injected prop?